plugins/module_utils/network/ios/rm_templates/l3_interfaces.py

```diff
@@ -47,6 +47,18 @@ def ip_tmplt(config_data):
     return cmd
 
 
+def ipv4_dhcp(config_data):
+    _data = config_data.get("ipv4", {}).get("dhcp")
+    if not _data.get("enable", True):
+        return None
```

Comment:
Is this really okay as None? I would expect this to be an empty string.

Reply:
Yep, empty string would be better